File: app/src/main/java/com/intelegain/agora/model/EmployeeProfile.java
package com.intelegain.agora.model;

/**
 * Created by maninder on 7/11/17.
 */


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class EmployeeProfile {

    @SerializedName("empid")
    @Expose
    public String empid;
    @SerializedName("empName")
    @Expose
    public String empName;
    @SerializedName("empAddress")
    @Expose
    public String empAddress;
    @SerializedName("empGender")
    @Expose
    public Object empGender;
    @SerializedName("empContact")
    @Expose
    public String empContact;
    @SerializedName("empJoiningDate")
    @Expose
    public String empJoiningDate;
    @SerializedName("empLeavingDate")
    @Expose
    public Object empLeavingDate;
    @SerializedName("empProbationPeriod")
    @Expose
    public Integer empProbationPeriod;
    @SerializedName("empNotes")
    @Expose
    public Object empNotes;
    @SerializedName("empEmail")
    @Expose
    public String empEmail;
    @SerializedName("empTester")
    @Expose
    public Boolean empTester;
    @SerializedName("empAccountNo")
    @Expose
    public String empAccountNo;
    @SerializedName("EmpPAN")
    @Expose
    public Object empPAN;
    @SerializedName("EmpUAN")
    @Expose
    public Object empUAN;
    @SerializedName("EmpEPF")
    @Expose
    public Object empEPF;
    @SerializedName("empBDate")
    @Expose
    public Object empBDate;
    @SerializedName("empADate")
    @Expose
    public String empADate;
    @SerializedName("empPrevEmployer")
    @Expose
    public Object empPrevEmployer;
    @SerializedName("empExperince")
    @Expose
    public Integer empExperince;
    @SerializedName("IsSuperAdmin")
    @Expose
    public Boolean isSuperAdmin;
    @SerializedName("IsAccountAdmin")
    @Expose
    public Boolean isAccountAdmin;
    @SerializedName("IsPayrollAdmin")
    @Expose
    public Boolean isPayrollAdmin;
    @SerializedName("IsPM")
    @Expose
    public Boolean isPM;
    @SerializedName("IsProjectReport")
    @Expose
    public Boolean isProjectReport;
    @SerializedName("IsProjectStatus")
    @Expose
    public Boolean isProjectStatus;
    @SerializedName("IsLeaveAdmin")
    @Expose
    public Boolean isLeaveAdmin;
    @SerializedName("IsActive")
    @Expose
    public Boolean isActive;
    @SerializedName("LocationFKID")
    @Expose
    public Integer locationFKID;
    @SerializedName("skillid")
    @Expose
    public Integer skillid;
    @SerializedName("InsertedOn")
    @Expose
    public Object insertedOn;
    @SerializedName("InsertedBy")
    @Expose
    public Integer insertedBy;
    @SerializedName("InsertedIP")
    @Expose
    public Object insertedIP;
    @SerializedName("ModifiedOn")
    @Expose
    public Object modifiedOn;
    @SerializedName("ModifiedBy")
    @Expose
    public Integer modifiedBy;
    @SerializedName("ModifiedIP")
    @Expose
    public Object modifiedIP;
    @SerializedName("IsTester")
    @Expose
    public Boolean isTester;
    @SerializedName("empPassword")
    @Expose
    public Object empPassword;
    @SerializedName("AnnualCTC")
    @Expose
    public Integer annualCTC;
    @SerializedName("CTC")
    @Expose
    public Integer cTC;
    @SerializedName("Gross")
    @Expose
    public Integer gross;
    @SerializedName("Net")
    @Expose
    public Integer net;
    @SerializedName("Resume")
    @Expose
    public Object resume;
    @SerializedName("Qualification")
    @Expose
    public Object qualification;
    @SerializedName("QualificationId")
    @Expose
    public Object qualificationId;
    @SerializedName("SecSkills")
    @Expose
    public Object secSkills;
    @SerializedName("SecSkillsId")
    @Expose
    public Object secSkillsId;
    @SerializedName("PrimarySkill")
    @Expose
    public Integer primarySkill;
    @SerializedName("Designation")
    @Expose
    public String designation;
    @SerializedName("Skill")
    @Expose
    public Object skill;
    @SerializedName("Type")
    @Expose
    public Object type;
    @SerializedName("Photo")
    @Expose
    public Object photo;
    @SerializedName("Mode")
    @Expose
    public Object mode;
    @SerializedName("EmpStatus")
    @Expose
    public Object empStatus;
    @SerializedName("LeavingStatus")
    @Expose
    public Object leavingStatus;
    @SerializedName("ProjectID")
    @Expose
    public Integer projectID;
    @SerializedName("SecurityLevel")
    @Expose
    public Integer securityLevel;
    @SerializedName("PrimarySkillDesc")
    @Expose
    public Object primarySkillDesc;
    @SerializedName("Event")
    @Expose
    public Object event;
    @SerializedName("ProfileID")
    @Expose
    public Integer profileID;
    @SerializedName("CAddress")
    @Expose
    public Object cAddress;
    @SerializedName("HistoryID")
    @Expose
    public Integer historyID;
    @SerializedName("ADUserName")
    @Expose
    public Object aDUserName;
    @SerializedName("Flag")
    @Expose
    public String flag;
    @SerializedName("strBirthday")
    @Expose
    public String strBirthday;

    @SerializedName("imageUrl")
    @Expose
    public String imageUrl;

}

File: app/src/main/java/com/intelegain/agora/model/EmployeeProfile.kt
package com.intelegain.agora.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by maninder on 7/11/17.
 */
class EmployeeProfile {
    @kotlin.jvm.JvmField
    @SerializedName("empid")
    @Expose
    var empid: String? = null
    @kotlin.jvm.JvmField
    @SerializedName("empName")
    @Expose
    var empName: String? = null
    @kotlin.jvm.JvmField
    @SerializedName("empAddress")
    @Expose
    var empAddress: String? = null
    @SerializedName("empGender")
    @Expose
    var empGender: Any? = null
    @kotlin.jvm.JvmField
    @SerializedName("empContact")
    @Expose
    var empContact: String? = null
    @kotlin.jvm.JvmField
    @SerializedName("empJoiningDate")
    @Expose
    var empJoiningDate: String? = null
    @SerializedName("empLeavingDate")
    @Expose
    var empLeavingDate: Any? = null
    @kotlin.jvm.JvmField
    @SerializedName("empProbationPeriod")
    @Expose
    var empProbationPeriod: Int? = null
    @SerializedName("empNotes")
    @Expose
    var empNotes: Any? = null
    @kotlin.jvm.JvmField
    @SerializedName("empEmail")
    @Expose
    var empEmail: String? = null
    @SerializedName("empTester")
    @Expose
    var empTester: Boolean? = null
    @kotlin.jvm.JvmField
    @SerializedName("empAccountNo")
    @Expose
    var empAccountNo: String? = null
    @SerializedName("EmpPAN")
    @Expose
    var empPAN: Any? = null
    @SerializedName("EmpUAN")
    @Expose
    var empUAN: Any? = null
    @SerializedName("EmpEPF")
    @Expose
    var empEPF: Any? = null
    @SerializedName("empBDate")
    @Expose
    var empBDate: Any? = null
    @kotlin.jvm.JvmField
    @SerializedName("empADate")
    @Expose
    var empADate: String? = null
    @SerializedName("empPrevEmployer")
    @Expose
    var empPrevEmployer: Any? = null
    @SerializedName("empExperince")
    @Expose
    var empExperince: Int? = null
    @SerializedName("IsSuperAdmin")
    @Expose
    var isSuperAdmin: Boolean? = null
    @SerializedName("IsAccountAdmin")
    @Expose
    var isAccountAdmin: Boolean? = null
    @SerializedName("IsPayrollAdmin")
    @Expose
    var isPayrollAdmin: Boolean? = null
    @SerializedName("IsPM")
    @Expose
    var isPM: Boolean? = null
    @SerializedName("IsProjectReport")
    @Expose
    var isProjectReport: Boolean? = null
    @SerializedName("IsProjectStatus")
    @Expose
    var isProjectStatus: Boolean? = null
    @SerializedName("IsLeaveAdmin")
    @Expose
    var isLeaveAdmin: Boolean? = null
    @SerializedName("IsActive")
    @Expose
    var isActive: Boolean? = null
    @SerializedName("LocationFKID")
    @Expose
    var locationFKID: Int? = null
    @SerializedName("skillid")
    @Expose
    var skillid: Int? = null
    @SerializedName("InsertedOn")
    @Expose
    var insertedOn: Any? = null
    @SerializedName("InsertedBy")
    @Expose
    var insertedBy: Int? = null
    @SerializedName("InsertedIP")
    @Expose
    var insertedIP: Any? = null
    @SerializedName("ModifiedOn")
    @Expose
    var modifiedOn: Any? = null
    @SerializedName("ModifiedBy")
    @Expose
    var modifiedBy: Int? = null
    @SerializedName("ModifiedIP")
    @Expose
    var modifiedIP: Any? = null
    @SerializedName("IsTester")
    @Expose
    var isTester: Boolean? = null
    @SerializedName("empPassword")
    @Expose
    var empPassword: Any? = null
    @SerializedName("AnnualCTC")
    @Expose
    var annualCTC: Int? = null
    @SerializedName("CTC")
    @Expose
    var cTC: Int? = null
    @SerializedName("Gross")
    @Expose
    var gross: Int? = null
    @SerializedName("Net")
    @Expose
    var net: Int? = null
    @SerializedName("Resume")
    @Expose
    var resume: Any? = null
    @SerializedName("Qualification")
    @Expose
    var qualification: Any? = null
    @SerializedName("QualificationId")
    @Expose
    var qualificationId: Any? = null
    @SerializedName("SecSkills")
    @Expose
    var secSkills: Any? = null
    @SerializedName("SecSkillsId")
    @Expose
    var secSkillsId: Any? = null
    @SerializedName("PrimarySkill")
    @Expose
    var primarySkill: Int? = null
    @kotlin.jvm.JvmField
    @SerializedName("Designation")
    @Expose
    var designation: String? = null
    @SerializedName("Skill")
    @Expose
    var skill: Any? = null
    @SerializedName("Type")
    @Expose
    var type: Any? = null
    @SerializedName("Photo")
    @Expose
    var photo: Any? = null
    @SerializedName("Mode")
    @Expose
    var mode: Any? = null
    @SerializedName("EmpStatus")
    @Expose
    var empStatus: Any? = null
    @SerializedName("LeavingStatus")
    @Expose
    var leavingStatus: Any? = null
    @SerializedName("ProjectID")
    @Expose
    var projectID: Int? = null
    @SerializedName("SecurityLevel")
    @Expose
    var securityLevel: Int? = null
    @SerializedName("PrimarySkillDesc")
    @Expose
    var primarySkillDesc: Any? = null
    @SerializedName("Event")
    @Expose
    var event: Any? = null
    @SerializedName("ProfileID")
    @Expose
    var profileID: Int? = null
    @SerializedName("CAddress")
    @Expose
    var cAddress: Any? = null
    @SerializedName("HistoryID")
    @Expose
    var historyID: Int? = null
    @SerializedName("ADUserName")
    @Expose
    var aDUserName: Any? = null
    @SerializedName("Flag")
    @Expose
    var flag: String? = null
    @kotlin.jvm.JvmField
    @SerializedName("strBirthday")
    @Expose
    var strBirthday: String? = null
    @kotlin.jvm.JvmField
    @SerializedName("imageUrl")
    @Expose
    var imageUrl: String? = null
}